zcash_test_vectors/transaction.py

```diff
@@ -476,22 +487,30 @@ def __init__(self, rand, have_orchard=True):
             # v^balanceOrchard is defined to be 0.
             self.valueBalanceOrchard = 0
 
-        assert is_coinbase == self.is_coinbase()
-
     def is_coinbase(self):
         # <https://github.com/zcash/zcash/blob/d8c818bfa507adb845e527f5beb38345c490b330/src/primitives/transaction.h#L969-L972>
         return len(self.vin) == 1 and bytes(self.vin[0].prevout.txid) == b'\x00'*32 and self.vin[0].prevout.n == 0xFFFFFFFF
 
-    def to_bytes(self, version_bytes, nVersionGroupId, nConsensusBranchId):
+    def to_bytes(self, version_bytes, version_group_id, consensus_branch_id):
         ret = b''
+        ret += self.header_bytes(version_bytes, version_group_id, consensus_branch_id)
+        ret += self.transparent_bytes(version_bytes)
+        ret += self.sapling_bytes(version_bytes)
+        return ret
```

Comment:
This seem to affect the test vectors but I don't see new vectors committed.

Reply:
No, they don't change the test vectors. Since the functions are called in place, the calls to `rand` remain in exactly the same order. 
(The CI checks for test vectors being changed, so it will fail if the test vectors are changed but not committed)